nextflow_schema.json

```diff
@@ -90,9 +90,10 @@
                 "input_region": {
```

Comment:
In some cases user a only interested in a specific genes or regions where they want to improve the resolution of their data.
They might not want to process the whole genome as it would be costly in time and resources. This csv file just need the chromosome name and start / end, that can be directly generated from the fasta file. A subworkflow is ready to do so, and simply providing "all", "chr1" or "chr21:100000-500000" a sufficient for this parameter.
But until `oneOf` is supported we will only focus on the csv file as suggested.